blog-cse/2023-01-19-content.md

```diff
@@ -0,0 +1,13 @@
+---
+title: January 13, 2023 - Content Release
```

Comment:
@ptobia Jan 19?

Reply:
Yes, sorry for the late submission.

Reply:
no worries -- just to confirm, you want to keep it as Jan 13?